lib/task/import/csvImportTask.class.php

```diff
@@ -219,6 +219,9 @@ public function execute($arguments = array(), $options = array())
       // Where to log errors to
       'errorLog' => $options['error-log'],
 
+      // Is the 'update' parameter set
+      'doUpdate' => ($options['update']) ? true : false,
```

Comment:
This will generate a PHP notice if the 'update' key doesn't exist in the $options array (it might not show up in AtoM--we sometimes suppress warnings and notices). I think also the parentheses are optional in PHP here (though in C if I recall they are required!). I'd do 'doUpdate' => isset($options['update']) ? true : false;